frontend/test/metabase/scenarios/question/new.cy.spec.js

```diff
@@ -255,7 +259,10 @@ describe("scenarios > question > new", () => {
         .should("contain", 1)
         .click();
 
```

Comment:
Let's please add `cy.wait("@dataset");` after each of these clicks.

[L:243](https://github.com/metabase/metabase/pull/21895/files#diff-7ea202f142e9881a22ed40fdf5aa77b70cc3a58b287d8674cb6d94fbd486cb4aR243)
[L:260](https://github.com/metabase/metabase/pull/21895/files#diff-7ea202f142e9881a22ed40fdf5aa77b70cc3a58b287d8674cb6d94fbd486cb4aR260)

Otherwise I'm pretty confident we'll see flakes sooner or later.

Reply:
Btw, you don't have to intercept the `/api/dataset`. That's already a part of the `openOrdersTable()` helper.